app/views/episodes/index.html.erb

```diff
@@ -1,14 +1,64 @@
-<p style="color: green"><%= notice %></p>
+<% content_for :title, t(".title") %>
```

Comment:
It would be nice if this actually had the current podcast's title in it (with a fallback if you're not within a podcast).  But we can tackle later.